src/main/java/com/google/api/codegen/transformer/MethodContext.java

```diff
@@ -51,9 +54,11 @@
 
   MethodContext cloneWithEmptyTypeTable();
 
-  /* Get the request type name and save it in the type table. */
-  String getAndSaveRequestTypeName();
+  String getTargetInterfaceFullName();
+
+  String getTargetInterfaceSimpleName();
+
+  String getInterfaceFileName();
 
-  /* Get the response type name and save it in the type table. */
-  String getAndSaveResponseTypeName();
+  String getGrpcContainerTypeName();
```

Comment:
Why isn't this in a grpc-specific class?

I'm guessing it's because the view model makes reference to some grpc names...